### PR TITLE
Feat: Introduce UNSTABLE_Divider component #DS-1302

### PR DESCRIPTION
**Pull request by crishpeen**

### CODE REVIEW

**Review by curdaj — APPROVED** [review]

File: .stylelintrc.js [hunk 1]
```diff
@@ -7,5 +7,15 @@ module.exports = {
     // This rule can lead to less readable code
     // @see: https://github.com/stylelint/stylelint-config-recommended/issues/14
     'declaration-block-no-redundant-longhand-properties': null,
+    // Allow UNSAFE_Component classnames
+    // @see: TODO link to the documentation
+    'selector-class-pattern': [
+      '(^(UNSAFE_)?([A-Z][a-zA-Z0-9]*)((--|__)[a-z][a-zA-Z0-9]*)*$)|(^([a-z][a-z0-9]*)(-[a-z0-9]+)*$)',
```
Comment (literat) [inline .stylelintrc.js:13]:
```suggestion
      '(^(unstable_)?([A-Z][a-zA-Z0-9]*)((--|__)[a-z][a-zA-Z0-9]*)*$)|(^([a-z][a-z0-9]*)(-[a-z0-9]+)*$)',
```

File: .stylelintrc.js [hunk 2]
```diff
@@ -7,5 +7,15 @@ module.exports = {
...
+      {
+        message:
+          'Expected class selector to be in format `MyComponent__myElement`, `MyComponent--modifier` ' +
+          'or `kebab-case` for utility classes (selector-class-pattern). Or `UNSAFE_MyComponent` for unsafe components.',
```
Comment (literat) [inline .stylelintrc.js:17]:
```suggestion
          'or `kebab-case` for utility classes (selector-class-pattern). Or `unstable_MyComponent` for unsafe components.',
```

File: packages/web-twig/src/Resources/components/UNSAFE_Divider/README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# UNSAFE Divider
+
+⚠️ This component is UNSAFE and there could be breaking changes in the future. Please use it with caution.
+
+Basic example usage:
+
+```twig
+<UNSAFE_Divider />
```
Comment (literat) [inline packages/web-twig/src/Resources/components/UNSAFE_Divider/README.md:8]:
```suggestion
<unstable_Divider />
```

Comment (crishpeen) [reply packages/web-twig/src/Resources/components/UNSAFE_Divider/README.md]: Actually, Twig needs it to be capitalized. So I decided to stick with UNSTABLE_Divider.

**Review by curdaj — APPROVED** [review]

File: .stylelintrc.js [hunk 4]
```diff
@@ -7,5 +7,15 @@ module.exports = {
     // This rule can lead to less readable code
     // @see: https://github.com/stylelint/stylelint-config-recommended/issues/14
     'declaration-block-no-redundant-longhand-properties': null,
+    // Allow UNSTABLE_Component classnames
+    // @see: https://github.com/lmc-eu/spirit-design-system/blob/main/docs/contributtion/experimental-code.md
+    'selector-class-pattern': [
+      '(^(UNSTABLE_)?([A-Z][a-zA-Z0-9]*)((--|__)[a-z][a-zA-Z0-9]*)*$)|(^([a-z][a-z0-9]*)(-[a-z0-9]+)*$)',
+      {
+        message:
+          'Expected class selector to be in format `MyComponent__myElement`, `MyComponent--modifier` ' +
```
Comment (literat) [inline .stylelintrc.js:16]: Is it possible to use a multiline template string, please, e. g. without the `+` string concatenation?

**Review by literat — APPROVED** [review]

File: packages/web/src/scss/components/UNSTABLE_Divider/_UNSTABLE_Divider.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+@use 'theme' as theme;
+
+.UNSTABLE_Divider {
+    display: block;
```
Comment (adamkudrna) [inline packages/web/src/scss/components/UNSTABLE_Divider/_UNSTABLE_Divider.scss:4]:
```suggestion
```

This is already provided by user agent:

image
image
image

File: tests/e2e/demo-components-compare.spec.ts-snapshots/unstable-divider-chromium-linux.png
Comment (adamkudrna) [inline tests/e2e/demo-components-compare.spec.ts-snapshots/unstable-divider-chromium-linux.png:1]: 🎉

**Review by adamkudrna — APPROVED** [review]
Nice! Let's just agree on the introductory message. Otherwise 👍🏻.

**Pull request merged**